trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapCoverEndDate.api-test.js

```diff
@@ -15,13 +14,13 @@ describe('mapCoverEndDate', () => {
 
     const result = mapCoverEndDate(day, month, year, {});
 
-    const coverEndDate = moment().set({
+    const coverEndDate =  set(new Date(), {
```

Comment:
If your linting is failing it might be here:

```suggestion
    const coverEndDate = set(new Date(), {
```